.env.example

```diff
@@ -1,6 +1,7 @@
 DATABASE_ENGINE=postgresql
 DATABASE_NAME=koku
+#DATABASE_NAME=test
 DATABASE_HOST=localhost
 DATABASE_PORT=15432
 DATABASE_USER=kokuadmin
-DATABASE_PASSWORD=''
+DATABASE_PASSWORD=kokuadmin
```

Comment:
Why are we changing values in the example file? It's an example, so it shouldn't be getting used for anything.

Reply:
Yes.  I'll revert this.  Didn't mean to check this in.

Reply:
I'll revert it.  The DATABASE_NAME however, might be helpful to keep since to run the unit tests the database name must be test.

Reply:
You should not rely on the example file for testing.

Reply:
@dccurtis I think an improvement on these lines that we could use is to set appropriate env variables in the tox.ini. I just kind of lazily pulled in everything through `passenv` which I was thinking of the travis use case primarily for. The next section is `setenv` where we could really use an override `DATABASE_NAME=test` could go there quite nicely. Likewise if I'm missing any others that need to get overridden for tests.